roxie/ccd/ccdserver.cpp

```diff
@@ -12290,6 +12290,7 @@ class CRoxieServerIndexWriteActivity : public CRoxieServerInternalSinkActivity,
             duplicateKeyCount = builder->getDuplicateCount();
             cummulativeDuplicateKeyCount += duplicateKeyCount;
             builder->finish(metadata, &fileCrc);
+            clearKeyStoreCache(false);
```

Comment:
Curious why this isn't specifically clearing the file just written (as in the hthor change) ?

Reply:
I address this in the commit comment:

For roxie, clear entire index key cache when a file is written (since the id's
stored in the cache do not seem to correspond to physical filenames, it's
non-trivial to clear just the one that is written.

Reply:
ah ok, missed that.